locales/en.default.schema.json

```diff
@@ -623,6 +623,9 @@
           "content": "Media",
           "info": "Learn more about [media types](https://help.shopify.com/manual/products/product-media)"
         },
+        "hide_variants": {
+          "label": "Hide unselected variants’ media on desktop"
```

Comment:
I think main-product is still using the old phrasing. I assume we want to update there as well?

Reply:
Hmm, actually, I don't think it should be applied there. The `on desktop` restriction only makes sense in the contect of the Featured Product section.

In the Product Page, the image is hidden both from the slider (mobile), the modal (desktop) and the main product gallery (desktop), so the less specific `Hide other variants' media after selecting a variant` still makes sense.